### PR TITLE
Create ping.md

### PR DESCRIPTION
**Pull request by BHLoneWolf0**

### CODE REVIEW

File: docs/commands/ping.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+layout: default
+title: Ping Command
+permalink: /docs/commands/ping/
+---
+
+# Ping command
+
+The `ping` command tells you about xf8b websocket and ping to discord server.
+
+**This is a level zero (no administrator needed) command.**
```
Comment (zAG6z) [inline docs/commands/ping.md:11]:
```suggestion
**Anyone can do this command.**
```

File: docs/commands/ping.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
+---
+layout: default
+title: Ping Command
+permalink: /docs/commands/ping/
+---
+
+# Ping command
+
+The `ping` command tells you about xf8b websocket and ping to discord server.
```
Comment (zAG6z) [inline docs/commands/ping.md:9]:
```suggestion
The `ping` command tells you about latency to the xf8b websocket and the latency to discord server.
```

**Review by zAG6z — CHANGES_REQUESTED** [review]

File: docs/commands/ping.md (hunk 1 above)
Comment (xf8b) [reply docs/commands/ping.md:11]:
```suggestion
**Anyone can use this command.**
```

File: docs/commands/ping.md (hunk 2 above)
Comment (zAG6z) [reply docs/commands/ping.md:9]: yes thanks for the touch up on my wording lmao

**Pull request merged**